Choose Your Own Emoji.md

```diff
@@ -98,13 +98,32 @@ Html.Events.onClick (Update.SetSelectedKey emoji)
 
 When this element is clicked, a `Msg` of `Update.SetSelectedKey String` will be triggered, which will eventually be consumed by our application.
 
-This is a good time to talk about the strange value of the `Msg` union type that seems to have the type `String` attached to the end of it. We've seen this pattern once before — in the `SetCurrentText String` value of `Update.Msg`.
+Like `SetCurrentText String`, `SetSelectedKey String` is a **tagged value**. In effect, we are saying that a `SetCurrentText` and `SetSelectedKey` are only valid values of `Msg` union type if they are accompanies by a string.
```

Comment:
I'm a little worried about overloading students in this section, but I do want to work in constructor functions / partial tagged values at some point.

For now, I'm going to pull this out into another issue, and merge what I have.